src/staticanalysis/bot/tests/test_reporter_phabricator.py

```diff
@@ -26,12 +27,13 @@
  - 1 defect found by clang-format
 
 You can run this analysis locally with:
+
  - `./mach clang-format -p path/to/file.cpp` (C/C++)
 
-For your convenience, here is a patch that fixes all the clang-format defects: https://diff.url (use it in your repository with `hg import` or `git apply`)
+For your convenience, [here is a patch]({results}/clang-format-PHID-DIFF-abcdef.diff) that fixes all the clang-format defects (use it in your repository with `hg import` or `git apply`)
 
 If you see a problem in this automated review, please report it here: https://bit.ly/2IyNRy2
-'''
+'''  # noqa
```

Comment:
Out of curiosity, what is the flake8 error here?

Reply:
Line 33 is really long